nvim-cmp-scm-1.rockspec

```diff
@@ -0,0 +1,37 @@
+local _MODREV, _SPECREV = "scm", "-1"
+rockspec_format = "3.0"
+package = "nvim-cmp"
+version = _MODREV .. _SPECREV
+
+description = {
+	summary = "A completion plugin for neovim",
+	labels = { "neovim" },
+	detailed = [[
+    A completion engine plugin for neovim written in Lua. Completion sources are installed from external repositories and "sourced".
+   ]],
+	homepage = "https://github.com/hrsh7th/nvim-cmp",
+	license = "MIT",
+}
+
+dependencies = {
+	"lua >= 5.1, < 5.4",
+}
+
+source = {
+	url = "http://github.com/hrsh7th/nvim-cmp/archive/v" .. _MODREV .. ".zip",
```

Comment:
Thank you for contribution.

This will be `http://github.com/hrsh7th/nvim-cmp/archive/vscm.zip`.
Is it expected?

Reply:
scm stands for `source control management`. development versions tend to be called like that. If you want to properly release a version e.g. "0.1" then you need to edit the rockspec to change the first line to `local _MODREV, _SPECREV = "scm", "-1"`. You dont have to version such a file (even though that could be handy) but what is useful is to upload it to luarocks.org so that other projects can reference it.